### DISCUSSION TITLE
scrypted can't login

### DISCUSSION THREAD

**Discussion by lebinho** [tteck/Proxmox#3798] [Help Needed] [answered]
Hi,
new to it but used as per instruction - installed via copy+paste from website. There were no errors.
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/scrypted.sh)"
`

looks like it finished installation on my proxmox with no issues and it is started. 
I tried `apt --fix-broken install` and I did install it again and also did reboot. No positive result.

I can ping the device with assigned static IP but they I try to open via browser i get 
_"This site can’t be reached"_

Other issue I'm experiencing is when i go to vm and to console.. asks for login so using default "root" and password "scrypted" returns _"Login incorrect"_

What can I do next ?

**Comment by tteck** [comment, marked as answer]
It didn't complete successfully if it's asking for credentials that you didn't set.

**Reply by lebinho** [reply]
There were no errors during installation. Will run again then.

**Reply by tteck** [reply]
It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Reply by lebinho** [reply]
I left it opened and running whole night on shell active. Re-installing.
